### ISSUE TITLE
[FEATURE] Create new command line argument for augment in __main__.py

### ISSUE THREAD

**Issue by 15bonte** [MouseLand/cellpose#657]
Hello,

Is there a reason why the _augment_ parameter used in _model.eval_ is set to False in __main__.py?

` 
out = model.eval(image, channels=channels, diameter=diameter,
                                do_3D=args.do_3D, net_avg=(not args.fast_mode or args.net_avg),
                                augment=False,
                                resample=(not args.no_resample and not args.fast_mode),
                                flow_threshold=args.flow_threshold,
                                cellprob_threshold=args.cellprob_threshold,
                                stitch_threshold=args.stitch_threshold,
                                min_size=args.min_size,
                                invert=args.invert,
                                batch_size=args.batch_size,
                                interp=(not args.no_interp),
                                normalize=(not args.no_norm),
                                channel_axis=args.channel_axis,
                                z_axis=args.z_axis,
                                anisotropy=args.anisotropy,
                                model_loaded=True)
`

I actually realized that setting it to True significantly improves my segmentation results, and I was surprised that there is no command line argument to change its value when calling cellpose from command line.

Would it be possible to add a new command line argument to enable the user to change it to True? Seems like it is just a few lines to change in the code, and may be very useful!

**Comment by carsen-stringer** [comment]
we can add it as a CLI argument, just out of curiosity what kind of images is it helping with?
